google/ggrc-core
# [Matrix view] QUICK-FIX Fix dropdown validation after save

# Issue description

Invalid dropdown validation after save answers or complete. 

# Steps to test the changes

1. Open Bulk Complete mode.
2. Add url to dp1, add url + comment to dp2.
3. Click "Save answers". "Edit info" button should disapear for dp1 and dp2.
4. Choose another value for dp2. Then previous saved value.
5. Click "Add info" button. Only comment should be shown in Required Info modal.

# Solution description

Save attached to assessment urls count, files count and update isCommentObligatoryToShow attribute after save answers.

# Sanity checklist




# PR Review checklist
